### ISSUE TITLE
Field labels are flipped on "Total cost, unit price"

### ISSUE THREAD

**Issue by GoogleCodeExporter** [evancharlton/android-mileage#7]
```
What steps will reproduce the problem?
1.
2.
3.

Expected behavior:

Actual behavior:

Mileage version:
Android version:
Device:

Please use labels and text to provide additional information.

```

Original issue reported on code.google.com by `evancharlton` on 27 Nov 2010 at 6:06

**Comment by GoogleCodeExporter** [comment]
```
Fixed in Mileage 3.0
```

Original comment by `evancharlton` on 30 Nov 2010 at 1:26
- Changed state: **Verified**

**Comment by GoogleCodeExporter** [comment]
Original comment by `evancharlton` on 17 Apr 2011 at 8:21
- Changed state: **Released**
